vite.config.ts

```diff
@@ -6,7 +6,6 @@ import tsconfigPaths from 'vite-tsconfig-paths';
 export default defineConfig({
   base: './',
   optimizeDeps: {
-    include: ['@endo/init'],
```

Comment:
```suggestion
    include: ['@endo/init'],
```

Suggesting to revert what i believe is a testing artifact

Reply:
Unrelated to lockdown, ignore suggestion